tests/server/blueprints/cases/test_cases_views.py

```diff
@@ -3,6 +3,7 @@
 from flask_login import current_user
 
 from scout.demo import delivery_report_path
+from scout.server.blueprints.cases import controllers
```

Comment:
Hm, did you actually use that in the end?

Reply:
Yes I'm using it in here https://github.com/Clinical-Genomics/scout/blob/f0cebdf09dd5aeb4607846267f24cb97f6c7c413/tests/server/blueprints/cases/test_cases_views.py#L369

and here: https://github.com/Clinical-Genomics/scout/blob/f0cebdf09dd5aeb4607846267f24cb97f6c7c413/tests/server/blueprints/cases/test_cases_views.py#L410

Reply:
Ah, the MME tests! Right! 👍